src/styles/globalStyles.ts

```diff
@@ -36,8 +38,8 @@ export default StyleSheet.create({
   },
   h4: {
     // fontFamily: 'DMSans-Regular',
-    fontSize: 16,
-    fontWeight: '500',
+    fontSize: 24,
+    fontWeight: '700',
     textAlign: 'left',
     color: 'black',
     lineHeight: 22,
```

Comment:
please undo this change and just add this text styling to your local styles. this is causing text on other screens to be the wrong size and weight.